hazelcast/sql.py

```diff
@@ -1481,14 +1495,15 @@ def parameters(self):
 
     @parameters.setter
     def parameters(self, parameters):
+        check_true(isinstance(parameters, list), "Parameters must be a list")
         if not parameters:
             self._parameters = []
         else:
             self._parameters = list(parameters)
```

Comment:
since we are sure that `parameters` is a list we can simplify here 
```suggestion
        self._parameters = parameters
```